test/integration/features/basic.feature

```diff
@@ -147,6 +147,12 @@ Feature: Basic
      """
      exposed
      """
+
+  Scenario: User can access ruby-ex route
```

Comment:
Got it. Also it looks more generic. Thanks hardy for the explanation. Will check the implementation same here.